crates/flox/src/commands/environment.rs

```diff
@@ -582,16 +594,32 @@ pub enum PushFloxmainOrEnv {
     },
 }
 
+#[derive(Debug, Clone, Bpaf)]
+enum PullSelect {
+    New {
+        /// Directory to create the environment in (default: current directory)
+        dir: Option<PathBuf>,
+        /// ID of the environment to pull
+        remote: EnvironmentRef,
```

Comment:
question nonblocking: should this be an optional positional? I think that's what we'll have for push which would be more consistent

Reply:
maybe, I'd like to circle back with Jenny/graham on the exact flags positionals, descriptions